cli/azd/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Release History
 
-## vNext (unreleased)
+## 1.0.3-beta.1 (unreleased)
+
+### Features Added
+
+- [[2364]](https://github.com/Azure/azure-dev/pull/2364) Display docker output during `package` and `deploy`.
+- [[2463]](https://github.com/Azure/azure-dev/pull/2463) Support `--docs` flag for all azd commands to show official documentation website.
 
```

Comment:
It should be there IMO

Reply:
I'd make sure @savannahostrowski highlights this one from the blog-post, and as part of all the world-of-azd.

We can add it here within the `Others` section, as it has no direct impact to the azd-binary.

I would also suggest to create a Changelog for the setup-azd.  

It would be more relevant if `azd` was able to generate the github workflow definitions automatically. In such case, we would be telling folks that azd would be generating a new pipeline definition.  However, there is no impact to the eyes of azd